vendor/TraceKit/tracekit.js

```diff
@@ -364,16 +364,16 @@ TraceKit.computeStackTrace = (function computeStackTraceWrapper() {
   // ex.stacktrace = n/a; see 'opera:config#UserPrefs|Exceptions Have Stacktrace'
 
   /**
-     * Computes stack trace information from the stack property.
-     * Chrome and Gecko use this property.
-     * @param {Error} ex
-     * @return {?Object.<string, *>} Stack trace information.
-     */
+   * Computes stack trace information from the stack property.
+   * Chrome and Gecko use this property.
+   * @param {Error} ex
+   * @return {?Object.<string, *>} Stack trace information.
+   */
   function computeStackTraceFromStackProp(ex) {
     if (typeof ex.stack === 'undefined' || !ex.stack) return;
 
     var chrome = /^\s*at (.*?) ?\(((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|[a-z]:|\/).*?)(?::(\d+))?(?::(\d+))?\)?\s*$/i,
-      gecko = /^\s*(.*?)(?:\((.*?)\))?(?:^|@)((?:file|https?|blob|chrome|webpack|resource|\[native).*?|[^@]*bundle)(?::(\d+))?(?::(\d+))?\s*$/i,
+      gecko = /^\s*(.*?)(?:\((.*?)\))?(?:^|@)((?:file|https?|blob|chrome|webpack|resource).*?:\/.*?|\[native code\]|[^@]*bundle)(?::(\d+))?(?::(\d+))?\s*$/i,
```

Comment:
pretty sure `blob` is `blob:foo` and not `blob://fo`

Reply:
Blob is `blob:http://url/for/some/endpoint`, which is matched by `blob.*?:\/`:) We have a test for it in place

Reply:
That depends on the browser. Current chrome adds the http origin to it, but `blob:UUID` is perfectly permissible and shows up in Firefox for instance.

You can try with `URL.createObjectURL(new Blob([new Int8Array()]))).toString()`

Reply:
Apparently blob urls are now supposed to always have an origin.  Since they are rare I might be okay with that but please add a comment why blob is okay.

Reply:
Added appropriate note and reformatted variables slightly so it's easier to move around.

Reply:
Also made it two separate commits so it's easier to track precise changes